### PR TITLE
[FIX]open form view corresponding to the type of invoice

### PR DESCRIPTION
**Pull request by Cedric-Pigeon**
After merging all selected invoices, the wizard shows the result in a tree view. 
Unfortunately, when you want to see it on the form view the supplier invoice form view is always loaded. Even if you work on customer invoices.

This PR suggest a fix for this bug.

### CODE REVIEW

File: account_invoice_merge/wizard/invoice_merge.py [hunk 1]
```diff
@@ -89,22 +89,21 @@ def merge_invoices(self):
...
-        }
+        xid = {
+            'out_invoice': 'action_invoice_tree1',
+            'out_refund': 'action_invoice_tree3',
+            'in_invoice': 'action_invoice_tree2',
+            'in_refund': 'action_invoice_tree4',
+        }[invoices[0].type]
+        action = aw_obj.for_xml_id('account', xid)
+        action.update({
+            'domain': [('id', 'in', ids + allinvoices.keys())],
```
Comment (pedrobaeza) [inline account_invoice_merge/wizard/invoice_merge.py:107]: Why are you getting ids? You only need `allinvoices.keys()`. You can use direct value writing instead an update on the dictionary.

Comment (Cedric-Pigeon) [inline account_invoice_merge/wizard/invoice_merge.py:107]: @pedrobaeza I am getting ids to be able to display in the tree view the new invoice and all original invoices cancelled in place of only the new one.

Comment (pedrobaeza) [inline account_invoice_merge/wizard/invoice_merge.py:107]: OK, I get it

**Comment by pedrobaeza** [comment]
:+1:

**Comment by Olivier-LAURENT** [comment]
:+1: Nice, user has now a quick visual validation on the result of its merge action ... !

**Comment by lmignon** [comment]
:+1:

**Pull request merged**